Engine/Engine.pro

```diff
@@ -21,7 +21,7 @@ TARGET = Engine
 TEMPLATE = lib
 CONFIG += staticlib
 CONFIG += moc
-CONFIG += boost boost-serialization-lib qt cairo python shiboken pyside 
+CONFIG += boost boost-serialization-lib egl wayland qt cairo python shiboken pyside
```

Comment:
will this break on systems that don't have EGL (eg macOS, Win)?

Reply:
Same as above.